src/handlers/outbound/consentRequests/{ID}/validate.ts

```diff
@@ -58,6 +58,7 @@ async function patch (_context: any, request: Request, h: StateResponseToolkit):
     key: consentRequestsRequestId,
     logger: h.getLogger(),
     thirdpartyRequests: h.getThirdpartyRequests(),
+    requestProcessingTimeoutSeconds: 3
```

Comment:
shouldn't we take this value from config?

Reply:
What do you suggest the default timeout be?

Reply:
going to set it to 30 akin to the sdk-scheme-adapter for now.